tests/tests_integration/test_api/test_data_modeling/test_containers.py

```diff
@@ -1,23 +1,33 @@
+from typing import Any
+
 import pytest
 
-import cognite.client.data_classes.data_modeling as dm
 from cognite.client import CogniteClient
+from cognite.client.data_classes.data_modeling import (
+    ContainerApply,
+    ContainerId,
+    ContainerList,
+    ContainerProperty,
+    Int32,
+    Space,
+    Text,
```

Comment:
I will just note that I have the opposite personal preference. By importing `from cognite.client import data_modeling as dm` it becomes clear where the classes comes from. In addition, you avoid having a lot imports at the top like this.

Reply:
Two reasons:
1) Consistency with rest of code base
2) It's important to me to have the imports at the top so that it is easy to see exactly what a given module depends on.

Reply:
Also when i use the `dm.BlaBla` notation, intellij doesn't flag type errors in docstring examples.

Reply:
Making that entirely consistent in https://github.com/cognitedata/cognite-sdk-python/pull/1266. Also running docstring tests so we catch more issues there.